go-ssb-bindings/vendor/go.cryptoscope.co/margaret/offset2/qry.go

```diff
@@ -259,7 +259,6 @@ func (qry *offsetQuery) fastFwdPush(ctx context.Context, sink luigi.Sink) (func(
 						break
 					}
 				}
-				sink.Close()
```

Comment:
So who closes the sink now?

Reply:
this was inconsistent between `fastFwdPush()` and _normal_ `Query()`.

The rational is that `io.Copy()` also doesn't close the writer because you might want to copy more things to it.